src/client/src/components/pages/courses/courseDetails/staticData.js

```diff
@@ -0,0 +1,67 @@
+export const state = {
+    courseName: "The Scaffolding Course",
+    category: "vocational",
+    startDate: "",
+    endDate: "",
+    description: "With thanks to Alandale who have provided the equipment, and Land Securities who have worked with us to facilitate the course, we are able to offer a further valuable, much sought after construction industry skill."
+};
+export const fields = [
+    [{
+            tag: "Input",
+            label: "course name",
+            name: "courseName",
+            width: "20rem",
+            placeholder: "course name"
+        },
+        {
+            tag: "DropDown",
+            label: "Course Category",
+            name: "category",
+            width: "20rem",
+            options: ['N/A', 'wawa']
+        }
+    ],
+    [
+        {
+            tag: "Input",
+            label: "Course start",
+            name: "startDate",
+            width: "20rem",
+            type: "date",
+        },
+        {
+            tag: "Input",
+            label: "Course end",
+            name: "endDate",
+            width: "20rem",
+            type: "date",
+        }
+    ],
+    [
+        {
+            tag: "Textarea",
+            label: "Description",
+            name: "description",
+            width: "45rem",
+            height: "10rem"
+        }
+    ],
+    [
+        {
+        tag: 'Button',
+        value: "Edit & Save",
+        color: "#272727",
+    },
```

Comment:
You need to add `staticField: true` to make the button appear in your form

Reply:
no ..
it is appear here